avm/utilities/pipelines/staticValidation/compliance/module.tests.ps1

```diff
@@ -397,11 +372,11 @@ Describe 'Module tests' -Tag 'Module' {
 
       if (Test-Path (Join-Path $moduleFolderPath 'tests')) {
 
-        # Can be removed after full migration to bicep test files
-        $moduleTestFilePaths = Get-ModuleTestFileList -ModulePath $moduleFolderPath | ForEach-Object { Join-Path $moduleFolderPath $_ }
+        # TODO: Can be removed after full migration to bicep test files
```

Comment:
Can't help but wonder if this is not a leftover from when moving away from json parameter files. Do you recall?

Reply:
Probably, I focused in this PR on that performance boost and bugfixes and did not look into the comments themselves.
I also think you added quite a few todo's a while back. Naturally, all out of scope of this PR, but something to tackle eventually.

Reply:
That's fair. Just wanted to double check as I saw the changed line. Let's tackle it together with the others, not blocking for me.